src/plopp/widgets/drawing.py

```diff
@@ -106,6 +106,7 @@ def remove_node(self, artist):
             self._destination.canvas.draw()
         output_node.remove()
         draw_node.remove()
+        del self._draw_nodes[nodeid]
```

Comment:
Could you just use `self._draw_nodes.pop(nodeid)` above instead of `__getitem__`?